reflection/Reflection/Reflection/attributes/ConfigurationItemAttribute.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+
+namespace Reflection.attributes
+{
+    [AttributeUsage(AttributeTargets.Property)]
+    public class ConfigurationItemAttribute : Attribute
+    {
+        public string SettingName { get; set; }
+        public Type ProviderType { get; set; }
+        
+        public string FilePath { get; set; }
+
+        public ConfigurationItemAttribute(string settingName, Type providerType, string filePath)
```

Comment:
make filePath optional parameter (set it to null if not provided), since you don't need the path if you use ConfigurationManagerConfigurationProvider